userspace/engine/filter_evttype_resolver.h

```diff
@@ -188,14 +189,15 @@ class filter_evttype_resolver
 private:
 	struct visitor : public libsinsp::filter::ast::expr_visitor
 	{
-		visitor(): m_expect_value(false) {}
+		visitor(): m_expect_value(false),m_inspector() {}
 		visitor(visitor&&) = default;
 		visitor& operator = (visitor&&) = default;
 		visitor(const visitor&) = default;
 		visitor& operator = (const visitor&) = default;
 
 		bool m_expect_value;
 		falco_event_types m_last_node_evttypes;
+		sinsp m_inspector;
```

Comment:
in the next step, we need to convert these methods to static ones, otherwise, we always have to allocate an inspector :/

Reply:
Agree!

Reply:
Totally agree!